moveit_ros/moveit_servo/include/moveit_servo/servo_node.h

```diff
@@ -45,18 +45,22 @@
 
 namespace moveit_servo
 {
-class ServoNode : public rclcpp::Node
+class ServoNode
```

Comment:
I think it's a very reasonable step to not inherit from `rclcpp::Node` anymore, even if it's recommended in some of the tutorials. This makes the interface much cleaner and allows us to more easily implement DI or middleware wrappers in the future.

Reply:
Would it make sense to provide a constructor that accepts a node as an argument `ServoNode()`? It seems a bit counter-intuitive to construct a node inside this class, but then depend on some external function spinning it.